### DISCUSSION TITLE
Fix some level design issues with "The Toilette Zone" of Bonus Island III

### DISCUSSION THREAD

**Discussion by ScriptForAll** [SuperTux/supertux#2696] [Ideas]
Hi! I've being playing Bonus Island III and noticed some issues with the level "The Toilette Zone", those being:

1. The scrolling of the background in section 3 being too fast that may cause motion sickness.
2. The burning sound is way too loud in section 4.

(Sorry if here is not the suitable place to discuss this)

**Comment by provigz** [comment]
This level is going to be removed, because of the Bonus Island remake in the next release (v0.7.0).

**Reply by ScriptForAll** [reply]
Is there any roadmap available for 0.7.0?

**Reply by provigz** [reply]
There is a Story Mode Roadmap on the wiki, however, v0.7.0 is also going to have bonus worlds remade, bonus islands being done with the help of the community.
